Google-Meet-Scheduler/README.md

```diff
@@ -0,0 +1,67 @@
+# Google Meet Scheduler
+
+![](http://ForTheBadge.com/images/badges/made-with-python.svg)
+
+The script [here](script.py) allows you to schedule a [google meeting](https://meet.google.com/) with multiple guests using [python](https://www.python.org/). You just need to provide the topic, time and list of email ids of the guests for the meeting and the [script](script.py) will handle the rest of the work.
+
+## Setup
+
+#### Step 1. You'll need to download a `credentials.json` file.
+
+- Go to [https://developers.google.com/calendar/quickstart/python](https://developers.google.com/calendar/quickstart/python).
+
+- Click the `Enable the Google Calendar API` button as shown below.
+<p align="center"><img height="250px" src="images/3.png"></p>
+
+- Click Next on each upcoming tab, **you don't need to change anything**.
+- Download the `credentials.json` file as shown below.
+<p align="center"><img height="200px" src="images/4.PNG"></p>
+
+- Move this file to the folder where your `script.py` resides.
+
+#### Step 2. Modify this part of the code as per your needs.
```

Comment:
you need to take these as inputs from the user, remove this step and delete the corresponding image